### DISCUSSION TITLE
Differential peaks comparisons between combined groups

### DISCUSSION THREAD

**Discussion by yxiao832** [GreenleafLab/ArchR#881] [Questions / Documentation]
Hi ArchR team,

If I want to get the differential peaks between cluster1,2,3 (as one combined group) and cluster4,5,6 (as one combined background group )

should I use the following command: 

markerTest <- getMarkerFeatures(
  ArchRProj = proj, 
  useMatrix = "PeakMatrix",
  groupBy = "Clusters",
  testMethod = "wilcoxon",
  bias = c("TSSEnrichment", "log10(nFrags)"),
  useGroups = c("Cluster1","Cluster2","Cluster3"),
  bgdGroups = c("Cluster4","Cluster5","Cluster6")
)

pma <- plotMarkers(seMarker = markerTest, name = c("Cluster1","Cluster2","Cluster3"), cutOff = "FDR <= 0.1 & abs(Log2FC) >= 1", plotAs = "MA")

Thank you very much!
Yang

**Comment by rcorces** [comment]
looks fine to me
